.golangci.yml

```diff
@@ -183,8 +195,7 @@ linters:
     # gosimple may suggest patterns that work only with more recent Go versions
     # - gosimple
     - nakedret
-    # formatting is disabled until we move to Go 1.13
-    # - gofmt
+    - gofmt
```

Comment:
Does this nullify the need for separate fmt checks in run-checks?

Reply:
let's double check, there's likely more linters we have in run-checks that may not be needed anymore

Reply:
The manual one in `run-checks` runs with `-s -d`, which prints out a nice diff, whereas the one in `golangci-lint` only supports `-s`. So the former is what we're used to and is nicer.

Regarding other lints, I did check, and some of the others in `run-checks` apply to all files, while the `golangci-lint` checks only seem to apply to `.go` files. So technically I think we want to preserve the superset of checks in `run-checks` in some cases.

I don't think we need `nakedret` in `run-checks` anymore, though.